services/venice-server/src/main/java/com/linkedin/venice/listener/ReadQuotaEnforcementHandler.java

```diff
@@ -218,6 +218,8 @@ public void channelRead0(ChannelHandlerContext ctx, RouterRequest request) {
      */
     TokenBucket tokenBucket = storeVersionBuckets.get(request.getResourceName());
     if (tokenBucket != null) {
+      // Can always emit the stale usage ratio first because it's using numbers from the previous refill
+      stats.recordReadQuotaUsageRatio(storeName, tokenBucket.getStaleUsageRatio());
```

Comment:
Why called it `stale usage ratio`? Can you elaborate?

Reply:
It's stale because it was calculated using previous refill interval's amortized requested QPS divided by refill amount. Depending on the refill interval it could be 30 seconds or a minute behind the current usage ratio.